src/css/_docs.scss

```diff
@@ -435,6 +435,7 @@
     font-size: 15px;
     line-height: 17px;
     position: relative;
+    border-top: 1px solid #333;
```

Comment:
Could the colour value be `transparent`? Should mean the `-active` colour can remain `$c-white` too.